test/receiverSpec.js

```diff
@@ -288,6 +303,28 @@ describe('Receiver Module', function () {
           expect(results).to.deep.equal(['sent'])
         })
 
+        it('should ignore transfers without packet.data', function * () {
```

Comment:
Actually I think it should be valid for a request to not include an expiry, but in this case the meat of the test is correct because it's really showing a case where the packet has been tampered with.